build-tools/enumification-helpers/enum-conversion-mappings.xml

```diff
@@ -1138,4 +1138,68 @@
   <map package='android.webkit' class='TracingConfig' prefix='CATEGORIES_' enum-name="Categories" api-level="28" />
   <map package='android.webkit' class='TracingConfig' fields='RECORD_CONTINUOUSLY RECORD_UNTIL_FULL' enum-name="TracingMode" api-level="28" />
 
+
+  <map package='android.app.admin' class='DevicePolicyManager.InstallSystemUpdateCallback' prefix='UPDATE_ERROR_' enum-name="UpdateErrorCode" api-level="29" />
+  <map package='android.app.admin' class='DevicePolicyManager' prefix='PRIVATE_DNS_MODE_' enum-name="PrivateDnsMode" api-level="29" />
+  <map package='android.app.admin' class='DevicePolicyManager' prefix='PRIVATE_DNS_SET_' enum-name="PrivateDnsSet" api-level="29" />
+  <map package='android.app.admin' class='DevicePolicyManager' prefix='PASSWORD_COMPLEXITY_' enum-name="PasswordComplexity" api-level="29" />
+  <map package='android.app' class='AppOpsManager' prefix='WATCH_FOREGROUND_' enum-name="WatchForeground" api-level="29" />
+  <map package='android.app' class='RemoteInput' prefix='EDIT_CHOICES_' enum-name="EditChoices" api-level="29" />
+  <map package='android.content.pm' class='ServiceInfo' prefix='FOREGROUND_SERVICE_' enum-name="ForegroundService" api-level="29" />
+  <map package='android.content.pm' class='PackageInstaller.SessionInfo' prefix='STAGED_SESSION_' enum-name="StagedSession" api-level="29" />
+  <map package='android.content.pm' class='PackageManager' prefix='INSTALL_REASON_' enum-name="InstallReason" api-level="29" />
+  <map package='android.content.pm' class='PackageManager' prefix='FLAG_PERMISSION_' enum-name="FlagPermission" api-level="29" />
+  <map package='android.graphics.fonts' class='FontStyle' prefix='FONT_SLANT_' enum-name="FontSlant" api-level="29" />
+  <map package='android.graphics' class='HardwareRenderer' prefix='SYNC_' enum-name="Sync" api-level="29" />
+  <map package='android.graphics' class='Paint' prefix='CURSOR_' enum-name="Cursor" api-level="29" />
+  <map package='android.graphics.text' class='LineBreaker' prefix='BREAK_STRATEGY_' enum-name="BreakStrategy" api-level="29" />
+  <map package='android.graphics.text' class='LineBreaker' prefix='HYPHENATION_FREQUENCY_' enum-name="HyphenationFrequency" api-level="29" />
+  <map package='android.graphics.text' class='LineBreaker' prefix='JUSTIFICATION_MODE_' enum-name="JustificationMode" api-level="29" />
+  <map package='android.hardware.biometrics' class='BiometricManager' prefix='BIOMETRIC_' enum-name="BiometricCode" api-level="29" />
+  <map package='android.hardware.camera2.params' class='RecommendedStreamConfigurationMap' prefix='USECASE_' enum-name="UsecaseMode" api-level="29" />
+  <map package='android.icu.lang' class='UScript' fields="*" enum-name="UScriptCode" api-level="24" />
```

Comment:
A few of these are not api-level 29.  Were these just enums that we have missed in the past?  Does adding them break our existing public API?  I think we will just mark the constant fields as `[Obsolete]`, but if we updated any methods using them that is a potential breakage.

Reply:
Thank @jpobst for your review.

All remaining-int-* lines were checked but they don't have any enum associated with it. Would definitely be nice to have a way to check it as verified and removed from the file. We should discuss some options to achieve it.

Regards the non api-level 29 items, they were showing as new lines (green on the diff) so I checked, found an enum and add it. You are bringing a good point related to break the api. What is the normal workflow on those cases? don't add the enum?

let me know your thoughts.